tests/baselines/reference/docker/office-ui-fabric.log

```diff
@@ -1,57 +1,2656 @@
+@uifabric/build:   create-package/plop-templates-react/just.config.ts:1:9
+@uifabric/build:     1 const { preset } = require('@uifabric/build');
+@uifabric/build:               ~~~~~~
+@uifabric/build:     'preset' was also declared here.
+@uifabric/build:   tasks/preset.ts:1:7
+@uifabric/build:     1 const preset = require('../just.config');
+@uifabric/build:             ~~~~~~
+@uifabric/build:     and here.
+@uifabric/build: create-package/plop-templates-node/just.config.ts:2:9 - error TS2451: Cannot redeclare block-scoped variable 'task'.
+@uifabric/build: 2 const { task } = just;
+@uifabric/build:           ~~~~
+@uifabric/build:   just.config.ts:3:9
+@uifabric/build:     3 const { task, series, parallel, condition, option, argv, addResolvePath, resolveCwd } = require('just-scripts');
+@uifabric/build:               ~~~~
+@uifabric/build:     'task' was also declared here.
+@uifabric/build: create-package/plop-templates-react/just.config.ts:1:9 - error TS2451: Cannot redeclare block-scoped variable 'preset'.
+@uifabric/build: 1 const { preset } = require('@uifabric/build');
+@uifabric/build:           ~~~~~~
+@uifabric/build:   create-package/plop-templates-node/just.config.ts:1:9
+@uifabric/build:     1 const { preset, just } = require('@uifabric/build');
+@uifabric/build:               ~~~~~~
+@uifabric/build:     'preset' was also declared here.
+@uifabric/build: create-package/plop-templates-react/src/version.ts:3:28 - error TS2307: Cannot find module '@uifabric/set-version' or its corresponding type declarations.
+@uifabric/build: 3 import { setVersion } from '@uifabric/set-version';
+@uifabric/build:                              ~~~~~~~~~~~~~~~~~~~~~~~
+@uifabric/build: gulp/plugins/util/getComponentInfo.ts:145:9 - error TS2322: Type 'Tag[]' is not assignable to type '{ title: string; description: string; type: null; name: string; }[]'.
+@uifabric/build:   Type 'Tag' is not assignable to type '{ title: string; description: string; type: null; name: string; }'.
+@uifabric/build:     Types of property 'type' are incompatible.
+@uifabric/build:       Type 'Type' is not assignable to type 'null'.
+@uifabric/build:         Type 'AllLiteral' is not assignable to type 'null'.
+@uifabric/build: 145         tags,
+@uifabric/build:             ~~~~
+@uifabric/build:   gulp/plugins/util/docs-types.ts:49:3
+@uifabric/build:     49   tags: {
+@uifabric/build:          ~~~~
+@uifabric/build:     The expected type comes from property 'tags' which is declared here on type 'ComponentProp'
+@uifabric/build: gulp/plugins/util/tsLanguageService.ts:25:18 - error TS2569: Type 'IterableIterator<string>' is not an array type or a string type. Use compiler option '--downlevelIteration' to allow iterating of iterators.
+@uifabric/build: 25       return [...files.keys()];
+@uifabric/build:                     ~~~~~~~~~~~~
+@uifabric/build: gulp/tasks/browserAdapters.ts:58:7 - error TS2554: Expected 1 arguments, but got 0.
+@uifabric/build: 58       resolve();
```

Comment:
Yes, this is expected because the `Promise` is untyped: https://github.com/microsoft/fluentui/blob/ea8ab5cbe8d27fec5ea30524a41bd4f2a26acdd6/scripts/gulp/tasks/browserAdapters.ts#L44